src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -11,6 +11,18 @@
 #include <ucs/sys/compiler.h>
 #include <arpa/inet.h> /* For htonl */
 
+#if HAVE_IBV_EXP_DM
+typedef struct uct_rc_mlx5_tm_dm_pack_data {
+    size_t     length;
+    const void *payload;
+} uct_rc_mlx5_tm_dm_pack_data_t;
+
+typedef struct uct_rc_mlx5_dm_pack_data {
```

Comment:
naming looks odd that  `uct_rc_mlx5_dm_pack_data` is inherited from `uct_rc_mlx5_tm_dm_pack_data`

Reply:
uct_rc_mlx5_dm_pack_data renamed to uct_rc_mlx5_am_dm_pack_data